<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestGraphicsViewClass</class>
 <widget class="QWidget" name="TestGraphicsViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestGraphicsView</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButtonAddText">
        <property name="text">
         <string>addText</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAddObj">
        <property name="text">
         <string>addObj</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonRefresh">
        <property name="text">
         <string>refresh</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditScale">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>100</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="MyGraphicsView" name="graphicsView"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>MyGraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="TestGraphicsView.qrc"/>
 </resources>
 <connections/>
</ui>
